public/index.html

```diff
@@ -49,18 +49,16 @@
 					<li>
 						<ul id="headerLinks" class="header-links">
 							
-								<li><a href="https://kubectl.docs.kubernetes.io/pages/app_management/introduction.html" class="header-link">kubectl Usage</a></li>
+								<li><a href="https://kubectl.docs.kubernetes.io/guides/introduction/" class="header-link">kubectl Usage</a></li>
```

Comment:
I know you didn't actually change this link now, but this attracted my attention to it and it seems like an odd place to link for "kubectl usage". What do you think about updating it to point here instead? https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/

Reply:
sure, updated